<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Okna komunikat√≥w</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="mywindow.qrc" >:/images/qt.png</iconset>
  </property>
  <widget class="QPushButton" name="pushButton_1" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>413</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>QMessageBox::about</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>50</y>
     <width>413</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>QMessageBox::critical</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>90</y>
     <width>413</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>QMessageBox::information</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>130</y>
     <width>413</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>QMessageBox::question</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>290</y>
     <width>391</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>320</y>
     <width>411</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>170</y>
     <width>413</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>QMessageBox::warning</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>210</y>
     <width>413</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>QMessageBox::AboutQt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>250</y>
     <width>413</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>QErrorMessage</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="mywindow.qrc" />
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX" >
   <number>10</number>
  </property>
  <property name="gridDeltaY" >
   <number>10</number>
  </property>
  <property name="gridSnapX" >
   <bool>true</bool>
  </property>
  <property name="gridSnapY" >
   <bool>true</bool>
  </property>
  <property name="gridVisible" >
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
